src/main/java/controller/KeyUtils.java

```diff
@@ -8,23 +8,34 @@
 import java.io.IOException;
 import java.io.UnsupportedEncodingException;
 import java.math.BigInteger;
+import java.net.URISyntaxException;
 
-import com.google.bitcoin.core.Base58;
-import com.google.bitcoin.core.ECKey;
-import com.google.bitcoin.core.ECKey.ECDSASignature;
-import com.google.bitcoin.core.Sha256Hash;
-import com.google.bitcoin.core.Utils;
+import org.bitcoinj.core.Base58;
+import org.bitcoinj.core.ECKey;
+import org.bitcoinj.core.ECKey.ECDSASignature;
+import org.bitcoinj.core.Sha256Hash;
+import org.bitcoinj.core.Utils;
 
 public class KeyUtils {
 
     final private static char[] hexArray = "0123456789abcdef".toCharArray();
-    final private static String PRIV_KEY_FILENAME = "bitpay_private.key";
-    
+    final private static String PRIV_KEY_FILENAME = "/bitpay_private.key";
+
     public KeyUtils() {}
-    
+
     public static boolean privateKeyExists()
     {
-        return new File(PRIV_KEY_FILENAME).exists();
+        return getPrivateKeyFile().exists();
```

Comment:
Small thing here, this is problematic if PRIV_KEY_FILENAME is not a resource. There would be an uncaught NPE. I think this is a good way to handle this (using resources), but we might want to set this up properly.